rr_iarrc/src/laplacian_line_detection/laplacian_line_detector.cpp

```diff
@@ -70,14 +75,15 @@ int main(int argc, char** argv) {
     std::string subscription_node;
     nhp.param("perfect_lines_min_cut", perfect_lines_min_cut, 200);
     nhp.param("Laplacian_threshold", Laplacian_threshold, 2);
+    nhp.param("adaptive_mean_threshold", adaptive_mean_threshold, 1);
 
     nhp.param("blockSky_height", blockSky_height, 220);
     nhp.param("blockWheels_height", blockWheels_height, 200);
     nhp.param("blockBumper_height", blockBumper_height, 200);
 
     nhp.param("subscription_node", subscription_node, std::string("/camera/image_color_rect"));
 
-    pub = nh.advertise<sensor_msgs::Image>("/Combined_lines", 1); //test publish of image
+    pub = nh.advertise<sensor_msgs::Image>("/lines_detection_img", 1); //test publish of image
     pub1 = nh.advertise<sensor_msgs::Image>("/Adpt_Thres_lines", 1);
     pub2 = nh.advertise<sensor_msgs::Image>("/Laplacian_lines", 1);
     pub3 = nh.advertise<sensor_msgs::Image>("/Colored_combined_lines", 1);
```

Comment:
As a future task, consider improving this node by making it a nodelet and using image_transport for publishing images. Ref http://www.clearpathrobotics.com/assets/guides/ros/Nodelet%20Everything.html